book/src/exec.md

```diff
@@ -0,0 +1,55 @@
+## On a high level
+Users use intents when they want to exchange their assets for some other asset. High level description of the flow:
```

Comment:
I'd say bartering/exchange of assets is a specific case of an application

Reply:
yeah. we should use more general language such as "enact a multiparty state transition"